.github/workflows/configs/kind-config.yaml

```diff
@@ -11,6 +11,25 @@ nodes:
       - |
         kind: KubeletConfiguration
         serverTLSBootstrap: true
+      - |
+        kind: ClusterConfiguration
+        scheduler:
+          extraArgs:
+            bind-address: "0.0.0.0"
+        apiServer:
+          extraArgs:
+            bind-address: "0.0.0.0"
+        controllerManager:
+          extraArgs:
+            bind-address: "0.0.0.0"
+        etcd:
+          local:
+            extraArgs:
+              listen-metrics-urls: "http://0.0.0.0:2381"
+      - |
+        kind: KubeProxyConfiguration
+        metricsBindAddress: "0.0.0.0:10249"
+
```

Comment:
😬 there is no such blank like before line 14... (duck for cover...)

Reply:
```suggestion
```